configs/BUILD

```diff
@@ -0,0 +1,13 @@
+package(default_visibility = ["//visibility:public"])
+
+genrule(
+    name = "example_configs",
+    srcs = glob(["*.json"]) + ["//examples:configs"],
+    outs = ["example_configs.tar"],
+    cmd = "./$(location configgen.sh) $(@D) $(locations //examples:configs)",
```

Comment:
I would remove the `./` from before `$(location ...`, since in theory `$(location)` could be an absolute path.